source/slang/hlsl.meta.slang

```diff
@@ -9614,7 +9634,17 @@ void TraceRay(
 {
     __target_switch
     {
-    case hlsl: __intrinsic_asm "TraceRay";
+    case hlsl:
```

Comment:
This function should now be [ForceInline] to keep resulting code clean.

Reply:
The reason ForceInline was not added is because it causes us to generate different code than glslang (cross-compile-test will fail). 

Understanding these tests are depreciated (and my new tests cover the code better) I will remove the cross-compile tests. I am bringing this up inside the PR for future reference and to explain why some tests will be modified